android/javatests/org/chromium/chrome/browser/BytecodeTest.java

```diff
@@ -804,6 +804,14 @@ public void testMethodsExist() throws Exception {
                         true,
                         int.class,
                         Context.class));
+        Assert.assertTrue(
+                methodExists(
+                        "org/chromium/chrome/browser/undo_tab_close_snackbar/UndoBarController",
+                        "showUndoBar",
+                        MethodModifier.REGULAR,
+                        false,
```

Comment:
We do need to check exact types when use reflection. Otherwise we may get runtime errors if something changes in the upstream.

Reply:
Done!